nashville-software-school/yak-25
# Login

## Link to Ticket
Closes #4 #2 

## Description of Proposed Changes

- should be able to login
- should be able to click remember me
- should error out if the credentials are wrong
- should be able to see dashboard once logged in
- should not be able to access navbar except logo(home) if not logged in

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout ask-nav-2```

click on login button
fill out the form 
- once with wrong credentials
- once with correct credentials
submit should take you to home page/ dashboard

## Impacted Areas in Application

List general components of the application that this PR will affect:

* Login
* Dashboard
* Nav

## Mentions @username

Tag users that need to review this code

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
